core/parachain/approval/approval_distribution.cpp

```diff
@@ -2774,10 +2822,12 @@ namespace kagome::parachain {
     REINVOKE(
         *main_pool_handler_, runDistributeApproval, vote, std::move(peers));
 
+    const common::Buffer enc{scale::encode(vote).value()};
```

Comment:
Would be good to move it into call of SL_DEBUG macro, to avoid useless computation in case logger doesn't accept debug-messages.